### ISSUE TITLE
[Test gap][Chassis] : Test case needed for show queue counters --voq

### ISSUE THREAD

**Issue by vmittal-msft** [sonic-net/sonic-mgmt#13365]
#### Steps to reproduce the issue:
1.
2.
3.

#### Describe the results you received:
Opening this Issue to track test gap for VoQ counters on the chassis.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by arlakshm** [comment]
@saksarav-nokia to check and close. The PRs for this issue are merged

**Comment by saksarav-nokia** [comment]
> [@saksarav-nokia](https://github.com/saksarav-nokia) to check and close. The PRs for this issue are merged

@arlakshm , The PR's crated for Issue   https://github.com/sonic-net/sonic-mgmt/issues/13366 are merged and covers the sonic mgmt to fetch the voq counters via sai-thrift . The PR https://github.com/sonic-net/sonic-mgmt/pull/16222 covers the cli side ( show queue counter --voq"). Once 16222 is merged, this issue can be closed.

**Comment by arlakshm** [comment]
`show queue counter --voq` support in test is present. The packet integrity counters is tracked in another issue. Closing this issue
